app/models/errors/httpErrors.scala

```diff
@@ -37,8 +37,9 @@ case class UnexpectedStatusError(code: Int, errorResponse: String) extends HttpE
     s"[UnexpectedStatusError]- RESPONSE status: $code, body: $errorResponse"
 }
 
-case object MultipleErrors extends HttpError {
-  override val message: String = "Received multiple errors."
+case class MultipleErrors(code: String, errorResponse: String) extends HttpError {
```

Comment:
Some of our errors have _code_ as a String and some have it as an Int. Do these need to be different or could we make them consistent?